.gitignore

```diff
@@ -34,3 +34,6 @@ setup.py
 .pdm.toml
 .pdm-python
 .pdm-build/
+
+# pyenv / rtx / asdf
+.tool-versions
```

Comment:
Do you use a global `.gitignore`? Personally I add these types of tool specific things in there and it covers me everywhere.

Not an issue adding this though, although maybe should move `.python-version` into this block too as it is for pyenv also.